### PR TITLE
Update migration docs

### PR DESCRIPTION
**Pull request by hankewyczz**
# Purpose

###### A brief description of the purpose of the PR's changes for the project.

Based on how the migration process went yesterday, these docs reflect new knowledge/clarification about the whole process.

### CODE REVIEW

File: docs/infrastructure/migration/migration.md [hunk 1]
```diff
@@ -1,102 +1,147 @@
...
+      - `sensitive`: `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are the same values as in GitHub
+      - `sensitive`: `CLOUDFLARE_EMAIL` is the email of someone with access to CloudFlare
+      - `sensitive`: `CLOUDFLARE_API_KEY` can be accessed by logging into CloudFlare using the email provided for `CLOUDFLARE_EMAIL`, go to `My Profile` > `API Tokens` > view the `Global API Key`
+      - `TF_LOG` should be set to the value `TRACE`
+        - This enables debug-level logging for Terraform runs. We'll use the output of these logs in a later step to create a more strict permissions-setting for our AWS IAM user.
+3.  Set the Terraform version
+    - In `Settings > General > Terraform Version`, set the version to the one specified in `providers.tf`
+
+aws_certificate_arn?
+aws_region?
```
Comment (megandouli) [inline docs/infrastructure/migration/migration.md:113]: just hardcoded to `us-east-1`

File: docs/infrastructure/migration/migration.md [hunk 2]
```diff
@@ -1,102 +1,147 @@
...
+      - `sensitive`: `GITHUB_TOKEN` is a GitHub personal access token you have to generate. [Instructions here](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+      - `sensitive`: `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are the same values as in GitHub
+      - `sensitive`: `CLOUDFLARE_EMAIL` is the email of someone with access to CloudFlare
+      - `sensitive`: `CLOUDFLARE_API_KEY` can be accessed by logging into CloudFlare using the email provided for `CLOUDFLARE_EMAIL`, go to `My Profile` > `API Tokens` > view the `Global API Key`
+      - `TF_LOG` should be set to the value `TRACE`
+        - This enables debug-level logging for Terraform runs. We'll use the output of these logs in a later step to create a more strict permissions-setting for our AWS IAM user.
+3.  Set the Terraform version
+    - In `Settings > General > Terraform Version`, set the version to the one specified in `providers.tf`
+
+aws_certificate_arn?
```
Comment (megandouli) [inline docs/infrastructure/migration/migration.md:112]: hmm interesting this variable doesn't look like it's used anywhere, I'm pretty sure Terraform creates the `aws_acm_certificate` in `acm.tf` so this should be safe to delete

**Review by megandouli — APPROVED** [review]
:chungus:

File: docs/infrastructure/migration/migration.md (hunk 2 above)
Comment (hankewyczz) [reply docs/infrastructure/migration/migration.md:112]:
```
Error: No value for required variable

  on variables.tf line 7:
   7: variable "aws_certificate_arn" {

The root module input variable "aws_certificate_arn" is not set, and has no
default value. Use a -var or -var-file command line argument to provide a
value for this variable.

Operation failed: failed running terraform plan (exit 1)
```

Comment (megandouli) [reply docs/infrastructure/migration/migration.md]: oh well you'd also need to remove the variable declaration in the terraform module

Comment (hankewyczz) [reply docs/infrastructure/migration/migration.md]: ah gotcha ty!

**Pull request merged**